README.md

```diff
@@ -104,6 +297,9 @@ generated inside Order.ts
 can contain a link to the swagger UI method reference, e.g. `http://example.com/swagger/swagger-ui.html#!/Order/Order`
 
 
+#### Usage of NGRX modules
```

Comment:
Not sure what does this say.

Reply:
changed to `NGRX workflow with generated modules, actions, effects, reducers and form services`